### PR TITLE
Seed val loss

### PR DESCRIPTION
**Pull request by piercus**

### CODE REVIEW

File: src/lbm/trainer/trainer.py [hunk 1]
```diff
@@ -169,7 +170,11 @@ def training_step(self, train_batch: Dict[str, Any], batch_idx: int) -> dict:
         }
 
     def validation_step(self, val_batch: Dict[str, Any], val_idx: int) -> dict:
-        loss = self.model(val_batch, device=self.device)["loss"]
+        if self.model.config.seed_key is not None and self.model.config.seed_key in val_batch:
+            seed = seed_from_string("-".join(val_batch[self.model.config.seed_key]))
```
Comment (deltheil) [inline src/lbm/trainer/trainer.py]:
```suggestion
        if self.model.config.seed_key is not None:
            assert self.model.config.seed_key in val_batch
            seed = seed_from_string("-".join(val_batch[self.model.config.seed_key]))
```

Comment (deltheil) [reply src/lbm/trainer/trainer.py]: WDYT?

**Review by deltheil — DISMISSED** [review]
LGTM (see my question though)

File: src/lbm/trainer/trainer.py (hunk 1 above)
Comment (piercus) [reply src/lbm/trainer/trainer.py]: Yes, it's better with the assert

**Review by deltheil — APPROVED** [review]
